HelloMinimal-CMake/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+# This is the most minimal version of a CMake file to compile an executable and
+# a library. Refer to HelloWorld-CMake for a more fleshed out example project.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(hello LANGUAGES Swift)
+
+add_library(HiKit hikit.swift)
+set_target_properties(HiKit PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_BINARY_DIR})
+add_executable(hello hello.swift)
```

Comment:
Add a newline before here to demarcate where the library handling and executable handling are.